### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Infinixius**
Update README with new title, shields.io badges, and code example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+  <img width=60% src="https://user-images.githubusercontent.com/68125679/193473284-e14ea33d-b086-4c28-870d-4221e766d775.png">
+  <br>
+  <a href="https://npmjs.com/package/oceanic.js"><img src="https://img.shields.io/npm/v/oceanic.js.svg?style=flat-square&color=informational"></a>
+  <img src="https://img.shields.io/github/stars/OceanicJS/Oceanic?color=yellow&style=flat-square">
+  <img src="https://img.shields.io/npm/dw/oceanic.js?color=red&style=flat-square">
+</p>
+
+```js
+const Oceanic = require("oceanic.js");
+const client = new Oceanic.Client({ auth: "Bot [TOKEN]" });
```
Comment (DonovanDMC) [inline README.md]:
```suggestion
const { Client } = require("oceanic.js");
const client = new Client({ auth: "Bot [TOKEN]" });
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+
+client.on("ready", async() => {
+    console.log("Ready as", client.user.tag);
+});
+
```
Comment (DonovanDMC) [inline README.md:16]:
```suggestion

client.on("error", (err) => {
    console.error("Something Broke!", err);
});

```

File: README.md [hunk 3]
```diff
@@ -1,6 +1,21 @@
...
+client.connect();
+```
 <hr>
```
Comment (DonovanDMC) [inline README.md:24]:
```suggestion

For more examples, see the [examples](https://github.com/OceanicJS/Oceanic/tree/dev/examples) folder on github.
<hr>
```

Github's being a pain in the ass, it won't let me have those 3 backticks up there in the suggestion

**Review by DonovanDMC — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ray-1337) [reply README.md]: maybe you can remove the Oceanic to `new Client`

**Review by ray-1337 — CHANGES_REQUESTED** [review]

**Review by DonovanDMC — APPROVED** [review]

**Pull request merged**